### PR TITLE
feat: add bubblewrap-suid

### LINKED ISSUE #11: bubblewrap-suid COPR
So bubblewrap-suid looks like a fork of bubblewrap. A possible way could be to just repackage the Arch build.

https://gitlab.archlinux.org/archlinux/packaging/packages/bubblewrap

https://archlinux.org/packages/extra/x86_64/bubblewrap-suid/

### PR DESCRIPTION
**Pull request by wandapeter**
Fixes #11 

Adds bubblewrap-suid COPR repository to install bubblewrap. replaces `/user/bin/bwrap` with suid bit set.

Allows to use Flatpak without reenabling user namespaces.

### CODE REVIEW

File: config/common-packages.yml [hunk 1]
```diff
@@ -22,6 +23,7 @@ install:
   - chkrootkit
   - koji
   - codium
+  - bubblewrap
```
Comment (RoyalOughtness) [inline config/common-packages.yml:26]: should this be bubblewrap-suid? the name will clash with the existing bubblewrap package

Comment (wandapeter) [reply config/common-packages.yml:26]: I thought replacing the package is the cleanest way.

While testing locally i used `override replace`  to replace the existing package. 

I tested this on a locally built image from `recipe-silverblue-main.yml` and it worked

But now if i think about it i am uncertain. But how would the dependencies be solved if we change the name to `bubblewrap-suid`?

Comment (wandapeter) [reply config/common-packages.yml:26]: I just figured that that Ublue treats these installs as overrides as well...

Output from the test VM seems good:
```bash
dev@fedora:~$ rpm -qa | grep bubblewrap
bubblewrap-0.8.0-1.fc39.x86_64
dev@fedora:~$ ls -l /usr/bin/bwrap 
-rwsr-xr-x. 3 root root 82536 Jan  1  1970 /usr/bin/bwrap
```

Comment (RoyalOughtness) [reply config/common-packages.yml:26]: > But now if i think about it i am uncertain. But how would the dependencies be solved if we change the name to `bubblewrap-suid`?

That's a fair point. This should be fine as-is and we can revisit if we run into problems later

**Comment by RoyalOughtness** [comment]
Apologies for the closure, I am rebasing the repo on top of startingpoint to get rid of the awful commit history. Better to do this now than after the project gains more traction. Please reopen this with the new base

**Pull request closed without merging**